apps/host/src/components/remote-mfe/remote-mfe.tsx

```diff
@@ -1,54 +1,115 @@
-import { component$, SSRStream, SSRStreamBlock } from '@builder.io/qwik';
+import {
+	component$,
+	SSRStream,
+	SSRStreamBlock,
+	StreamWriter,
+	useClientEffect$,
+	useSignal,
+} from '@builder.io/qwik';
 import { useLocation } from '@builder.io/qwik-city';
 import type { RemoteData } from '../../../../../libs/shared/remotes';
 
 export interface Props {
 	remote: RemoteData;
+	onlyFetchOnScroll?: true;
 }
 
-export default component$(({ remote }: Props) => {
-	const localtion = useLocation();
+export default component$(({ remote, onlyFetchOnScroll }: Props) => {
```

Comment:
I think `onlyFetchOnScroll` is ok, but I wonder if there could be a better name for the prop, any idea?

Reply:
Thanks for the great refactor. 
_fetchOnScoll_ sounds better because if it's false the component is fetched immediately.
I mean, _only _ is a weird word 🤔 
does that make sense?

Reply:
thanks for the kind words :smiley: 

yeah anyways I do agree `only` is unnecessary there and it makes things more confusing then not

I'll remove it, thanks for the suggestion :smile: